README.md

```diff
@@ -90,7 +76,7 @@ dotnet ef migrations script -i --project ../PolloPollo.Entities -o <FILE>
         },
     }
     ```
-    2. ```> dotnet publish``` - This command builds the code and creates the directory with the builded dll's at the path: PolloPollo.Web/bin/Debug/netcoreapp2.2/publish
+    2. ```dotnet publish``` - This command builds the code and creates the directory with the builded dll's at the path: PolloPollo.Web/bin/Debug/netcoreapp2.2/publish
```

Comment:
Not correct path

Reply:
Maybe we should use `dotnet publish -c Release` ?

Reply:
Should be fixed now.